Source/engine/displacement.hpp

```diff
@@ -0,0 +1,111 @@
+#pragma once
+
+#include <cmath>
+
+#include "direction.hpp"
+#include "utils/stdcompat/abs.hpp"
+
+namespace devilution {
+
+struct Displacement {
+	int deltaX;
+	int deltaY;

```

Comment:
The `delta` prefix seems redundant here, it's already a `Displacement`?

Reply:
I think the prefix makes sense and adds semantic to usages. The value itself should provide semantic to what it is without necessarily depending on the parent class/struct in some cases.